smart-contracts/scripts/helpers/utils.js

```diff
@@ -93,10 +108,37 @@ function cacheBuster(url) {
   return finalUrl;
 }
 
+/**
+ * Removes duplicates from arrays
+ * @param {array} list Your array
+ * @returns {array} an array containing no duplicates
+ */
+function removeDuplicates(list) {
+  const uniqueSet = new Set(list);
+  return [...uniqueSet];
+}
+
+/**
+ * Compares two arrays and returns true if they have the exact same elements, even if out of order
+ * @param {array} List 1
+ * @param {array} List 2
+ * @returns {bool} Do they have the same elements?
+ */
+function hasSameElements(a, b) {
+  if (a.length !== b.length) return false;
```

Comment:
Javascript/Typescript Set operators are awful, they dont support basic operations like intersection, union, and difference. We should probably code those in this utils.js file and then have hasSameElements check to see if the intersection of the two sets is equal to either set, rather then doing a nested for loop.